kie-server-parent/kie-server-remote/kie-server-client/src/main/java/org/kie/server/client/impl/UserTaskServicesClientImpl.java

```diff
@@ -828,17 +828,23 @@ public TaskInstance findTaskByWorkItemId(Long workItemId) {
     }
 
     @Override
-    public TaskInstance findTaskById(Long taskId) {
+    public TaskInstance findTaskById(Long taskId){
+        return findTaskById(taskId, false);
+    }
+
+    @Override
+    public TaskInstance findTaskById(Long taskId, boolean includeSLA) {
```

Comment:
Not sure if we need 2 ways of getting a  `TaskInstance` without SLA (either by calling `findTaskById(taskId)` or `findTaskById(taskId, false)`). I understand that this way we can reuse the implementation, but we can also do `findTaskById` and `findTaskByIdWithSLA` and provide one private method as implementation. Just a suggestion, I don't want to be strict too much :)

Reply:
Now I get it, that by doing that you can use just `client.findTaskById(taskId, includeSLA);` in `RemoteTaskServiceImpl#getTask`. So I guess it is just a matter of whether we want to make it more pleasant in our UI component or in the KIE Server client API.

Reply:
@MarianMacik  there's no problem on changing our UI services, the only thing that stopped me was preserve the API in order to maintain the backwards compatibility. As you prefer, we can keep just the findTaskById(taskId,boolean) o keep both methods

Reply:
Yes, there is no doubt we have to preserve the backwards compatibility, but it will be preserved even with 2 separate methods since the first one (`findTaskById(taskId)`) is already there. We would just add the second one, i.e. `findTaskByIdWithSLA(taskId)`. We can also let other reviewers to express their opinions. Wdyt @mswiderski @cristianonicolai ?

Reply:
@MarianMacik sorry I misunderstood The point is where to do the check. If you think is more clear create the two methods instead pass the boolean, is good to me. ( Believe me during the development I pass throw having both, but I decided pass the boolean). Let me change it

Reply:
It was just an idea :) I find more safe to have an API with two methods like `findTaskById(taskId)` and `findTaskByIdWithSLA(taskId)` - I think it is more readable than to have something like `findTaskById(taskId)` and `findTaskById(taskId, true/false)`.

But as I said, we don't need to implement. I would wait for other guys to say their opinion. I don't want to be nitpicking.

Reply:
updated